app/responsive-demo/page.tsx

```
@@ -0,0 +1,474 @@
+                      <Button>Standard Size</Button>
+                    </div>
+                    
+                    <div>
+                      <p className="text-sm text-muted-foreground mb-2">Touch-Target Button</p>
+                      <Button className="touch-target">Touch-Friendly</Button>
+                    </div>
+                    
+                    <ShowOnMobile>
+                      <div className="bg-muted p-4 rounded mt-6">
+                        <p className="text-sm">
+                          On mobile devices, it's especially important to have large enough touch targets.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)